pycbc/events/stat.py

```diff
@@ -963,6 +963,13 @@ def __init__(self, sngl_ranking, files=None, ifos=None, **kwargs):
         # This will be used to keep track of the template number being used
         self.curr_tnum = None
 
+        # This applies a constant offset to *all* statistic values. Can be
```

Comment:
Out of interest is this degenerate with 'benchmark_lograte' which shifts all the log noise rate stat terms by a constant?

Reply:
Yes it is - that was designed to approximately "try to make 0 correspond to a 50/50 chance of being real"